telegram/bot.py

```diff
@@ -4371,6 +4373,89 @@ def set_my_commands(
 
         return result  # type: ignore[return-value]
 
+    @log
+    def copy_message(
+        self,
+        chat_id: Union[int, str],
+        from_chat_id: Union[str, int],
+        message_id: Union[str, int],
+        caption: str = None,
+        parse_mode: str = None,
+        caption_entities: SLT[MessageEntity] = None,
```

Comment:
that's a good idea 👍  we should do that for ILQR and IM*, too

Reply:
On second thought, if we allow singles here, we need to handle that properly below. Maybe it's better to only allow `Union[Tuple[ME, ...], List[ME]]`

Reply:
oh right forgot about SLT also allowing singles